README.md

```diff
@@ -16,7 +16,7 @@ Remote is meant to allow people from anywhere (or at least anywhere in a given t
 | AirHouse | All | Full remote | [Link](https://www.airhouse.io/careers) | ? |
 | Aha! Labs | All | Regional remote | [Link](https://www.aha.io/company/careers/current-openings) | ? |
 | Alation | All | Full remote depending on the position | [Link](https://www.alation.com/careers/) | ? |
-| Angle Health | All | Full remote | [Link](https://jobs.lever.co/AngleHealth) | ? |
+| Angle Health | All | Full remote | [Link](https://www.anglehealth.com/careers) | ? |
```

Comment:
This must be changed to `Some` (instead of `All`) since some positions are hybrid.